llvm/include/llvm/Config/llvm-config.h.cmake

```diff
@@ -202,6 +202,10 @@
 #cmakedefine LLVM_HAS_LOGF128
 
 /* Define if building LLVM with LLVM_BUILD_TELEMETRY */
+/* DEPRECATED - use LLVM_ENABLE_TELEMETRY */
```

Comment:
Do we need to support the old define? I don't see anything else like that in this file. I'd just remove it.

Reply:
@rupprecht Hi, do you know if there's any other place (bazel builds?) that need updating? Or can I just remove the flag directly w/o deprecating it? Thanks!